test

```diff
@@ -581,7 +581,7 @@ function receiver_name_pass {
 }
 
 function commit_title_pass {
-	git log --oneline "$(git merge-base HEAD master)"...HEAD | while read -r l; do
+	git log --oneline "$(git merge-base HEAD $(git rev-parse --abbrev-ref --symbolic-full-name @{u}))"...HEAD | while read -r l; do
```

Comment:
Latest test failure:

```
test:584:43: warning: Quote this to prevent word splitting. [SC2046]
test:584:94: warning: This { is literal. Check expression (missing ;/
?) or quote it. [SC1083]
test:584:96: warning: This } is literal. Check expression (missing ;/
?) or quote it. [SC1083]
```
We might need to do something like:

```
gitBranch=$(git rev-parse --abbrev-ref --symbolic-full-name "@{u}")
gitBaseHash=$(git merge-base HEAD "${gitBranch}")
git log --oneline "${gitBaseHash}"...HEAD | while read -r l; do
```
 
Fair warning: I haven't run this. It needs to be tested with `shellcheck`.